src/customer/customer-request-sender.ts

```diff
@@ -10,6 +11,13 @@ export default class CustomerRequestSender {
         private _requestSender: RequestSender
     ) {}
 
+    updateCustomer(customer: CurrentCustomer, { timeout }: RequestOptions = {}): Promise<Response<CurrentCustomer>> {
+        const url = `/api/storefront/customer`;
```

Comment:
Just use regular `''` instead of back ticks if there's no string interpolation. 🍹 💅